app/Http/Controllers/IncomingTextController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\TwilioRequest as Request;
+
+class IncomingTextController extends Controller
+{
+    protected $commands = [
+        "unsubscribe" => [
```

Comment:
Twilio handles the blacklisting automatically and then they send the message through so that we can act on it, I didn't see any specific webhook setup but the link in the pr explains how it works.